Contest_API.md

```diff
@@ -252,17 +252,19 @@ absolute timestamps.
     with properties as defined below.
   - Arrays (type **`array of <type>`** in the specification) are built-in JSON 
     arrays of some type defined above.
+  - Nullable types (type **`<type> ?`** in the specification) are either a
+    value of a type defined above, or `null`.
 
 Properties for file reference objects:
 
-| Name     | Type    | Nullable? | Description
-| -------- | ------- | --------- | -----------
-| href     | string  | yes       | URL where the resource can be found. Relative URLs are relative to the `baseurl`. Must point to a file of intended mime-type. Resource must be accessible using the exact same (possibly none) authentication as the call that returned this data.
-| filename | string  | no        | POSIX compliant filename. Filenames must be unique within the endpoint object where they are used. I.e. an organization can have (multiple) `logo` and `country_flag` file references, they must all have a different filename, but different organizations may have files with the same filename.
-| hash     | string  | yes       | MD5 hash of the file referenced.
-| mime     | string  | no        | Mime type of resource.
-| width    | integer | depends   | Width of the image. Required for files with mime type image/*.
-| height   | integer | depends   | Height of the image. Required for files with mime type image/*.
+| Name     | Type      | Description
+| -------- | --------- | -----------
+| href     | string ?  | URL where the resource can be found. Relative URLs are relative to the `baseurl`. Must point to a file of intended mime-type. Resource must be accessible using the exact same (possibly none) authentication as the call that returned this data.
+| filename | string    | POSIX compliant filename. Filenames must be unique within the endpoint object where they are used. I.e. an organization can have (multiple) `logo` and `country_flag` file references, they must all have a different filename, but different organizations may have files with the same filename.
+| hash     | string ?  | MD5 hash of the file referenced.
+| mime     | string    | Mime type of resource.
+| width    | integer ? | Width of the image. Required for files with mime type image/*.
+| height   | integer ? | Height of the image. Required for files with mime type image/*.
```

Comment:
I just realized that this means width and height can be either an integer or 'null', and I don't like that. :) In the case of integers or booleans IMHO it's preferable to leave the value out and there is a default specified ('unknown' in this case).